Tutorials/whatiswot/website/docs/wot/application-domains/industrial-and-agriculture-systems.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_label: Industrial and Agriculture Systems
+---
+
+# Industrial and Agriculture Systems
+
+![coming-soon](/img/tutorial/What-Is-Wot/coming_soon_banner.png)
+
+The Web of Things finds applications in diverse fields beyond smart homes and cities. This tutorial will focus on industrial and agriculture systems - two major and complex examples.
+
+## Industrial systems
+
+Unlike smart home scenarios, industrial applications have broader and more demanding requirements. These include advanced monitoring of complex connected manufacturing equipment, tracking products and predicting possible machine failures.
+
+> In factories, for example, devices from multiple manufacturers must work together seamlessly. WoT simplifies this by supporting old and new industrial protocols alike, such as Modbus, Profinet, OPC UA, and MQTT. Each device is described using a Thing Description, which tells other systems exactly how to interact with it. TDs enhance existing workflows by reducing complexity and engineering effort without introducing overhead or security concerns.
```

Comment:
This should be moved after the next paragraph as a concrete example and should be better linked to the figure